src/DynamoCoreWpf/ViewModels/Watch3D/AttachedProperties.cs

```diff
@@ -435,6 +377,42 @@ public static bool IsSpecialRenderPackage(DependencyObject element)
 
         //#endregion
 
+        #region utils
+        
+        /// <summary>
+        /// Sets all colors on Geometry to a single color.
+        /// </summary>
+        /// <param name="geom"></param>
+        /// <param name="color"></param>
+        private static void SetAllColors(GeometryModel3D geom, Color4 color)
+        {
+            var newColorCollection = new Color4Collection(Enumerable.Repeat(color, geom.Geometry.Colors.Count));
+            geom.Geometry.Colors = newColorCollection;
+        }
+
+        /// <summary>
+        /// Sets alpha channel of existing colors to given value, optionally attempts to reset all colors
+        /// first to original colors from renderpackages.
+        /// </summary>
+        /// <param name="geom"></param>
+        /// <param name="alpha"></param>
+        /// <param name="resetColorsFirst"></param>
```

Comment:
This is minor but can you set the description for the params as well, especially this boolean?

Reply:
will do.

Reply:
done.